api/proto/config/storage.proto

```diff
@@ -73,9 +67,6 @@ message Image {
   string sha256 = 3; // EVE will verify this sha for the content
   Format iformat = 4;
 
-  // if its signed image
-  SignatureInfo siginfo = 5;
```

Comment:
You need to leave a comment in place so we don't accidentally reuse tag 5.
A format we use elsewhere is
// deprecated 5; SignatureInfo siginfo